lib/deaf_grandma.rb

```diff
@@ -19,9 +19,15 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    counter = 0
+    if input == input.downcase
+      return "SPEAK UP SONNY!"
+    elsif input == input.upcase && input != "BYE"
+      return "NOT SINCE 1964!"
+    elsif input == input.upcase && input == "BYE"
+      return  "SEE YOU LATER SONNY!"
+      exit
```

Comment:
Is this exit line ever run? It seems like it wouldn't be since you return on the line right before